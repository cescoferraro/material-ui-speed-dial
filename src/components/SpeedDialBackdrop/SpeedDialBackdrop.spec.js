import React from 'react';
import { shallow } from 'enzyme';
import { SpeedDialBackdrop } from './SpeedDialBackdrop';
import styles from './SpeedDialBackdrop.styles';

const props = {
	classes: styles,
	className: 'className',
	state: 'opened',
	onClick() {},
};

describe('SpeedDialBackdrop', () => {
	it('snapshot: default', () => {
		const _props = { ...props };
		const tree = shallow(<SpeedDialBackdrop {..._props} />);
		expect(tree).toMatchSnapshot();
	});
	it('styles', () => {
		expect(
			typeof styles({
				zIndex: { appBar: 1000 },
				palette: { grey: { 200: '' } },
			})
		).toBe('object');
	});
});
