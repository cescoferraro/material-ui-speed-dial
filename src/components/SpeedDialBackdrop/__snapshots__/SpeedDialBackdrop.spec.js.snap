// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SpeedDialBackdrop snapshot: default 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown
    className="className"
    classes={[Function]}
    onClick={[Function]}
    state="opened"
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "className": "undefined undefined className",
      "onClick": [Function],
    },
    "ref": null,
    "rendered": null,
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "className": "undefined undefined className",
        "onClick": [Function],
      },
      "ref": null,
      "rendered": null,
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
